src/Client.php

```diff
@@ -467,7 +467,7 @@ protected function createTransaction(array $options = null, $callable = null)
      * Creates a new publish/subscribe context and returns it, or starts its loop
      * inside the optionally provided callable object.
      *
-     * @param mixed ... Array of options, a callable for execution, or both.
+     * param mixed ...$arguments Array of options, a callable for execution, or both.
```

Comment:
This is wrong, missing `@`. Several times in other files too.

Reply:
Yes, I've removed it from non-existent parameters.

in the next PR they will be dealt with: https://github.com/predis/predis/pull/1025#issuecomment-1396075402

Reply:
Should I include it in this PR?

Reply:
Yes, I don't want to merge broken phpdocs.

Reply:
Continuing with 🍊🍊🍊 power ...